torch/fx/symbolic_trace.py

```diff
@@ -123,9 +124,8 @@ def trace(self, root: torch.nn.Module) -> Graph:
         self.root = root
         fn = type(root).forward
         self.graph = Graph()
-
         assert isinstance(fn, FunctionType)
-        co = fn.__code__
+        co = AST_Rewriter().rewrite(fn)
```

Comment:
Can you make a subclass of `Tracer` with this behavior rather than modifying this here? Simply because this is experimental and we want to keep it opt-in for now